src/axom/quest/examples/scattered_interpolation.cpp

```diff
@@ -0,0 +1,788 @@
+// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+/*!
+ * \file scattered_interpolation.cpp
+ * \brief This examples uses a Delaunay mesh to perform scattered data
+ * interpolation on fields defined over a point set
+ */
+
+#include "axom/core.hpp"
+#include "axom/slic.hpp"
+#include "axom/slam.hpp"
+#include "axom/sidre.hpp"
+#include "axom/primal.hpp"
+#include "axom/quest.hpp"
+
+#include "conduit.hpp"
+#include "conduit_blueprint.hpp"
+#include "conduit_relay.hpp"
+
+#include "axom/fmt.hpp"
+#include "axom/CLI11.hpp"
+
+#include <memory>
+
+namespace primal = axom::primal;
+namespace quest = axom::quest;
+namespace sidre = axom::sidre;
+
+namespace internal
+{
+namespace blueprint
+{
+/**
+ *  \brief Simple wrapper to a blueprint particle mesh
+ *
+ *  Given a sidre Group, creates the stubs for a mesh blueptint particle mesh
+ */
+struct PointMesh
```

Comment:
This `PointMesh` class is very similar to the one in the distributed closest point example: https://github.com/LLNL/axom/blob/bd43ea33a14608a72519cf64bdea7143ac32ea38/src/axom/quest/examples/quest_distributed_distance_query_example.cpp#L131-L395

I'd like to combine these and move them into a `sidre/blueprint` directory that simplifies processing of mesh blueprint hierarchies.

I'll create an issue for this as well.